src/main/java/uk/gov/companieshouse/scanupondemand/orders/api/model/ScanUponDemandItem.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.companieshouse.scanupondemand.orders.api.model;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * An instance of this represents a scan upon demand item.
+ */
+
+@Document(collection = "scan-upon-demands")
```

Comment:
should this be snake case?

Reply:
correct me if I am wrong, it doesn't look like it is in the certified-copies or certificates APIs

Reply:
So I'm looking at certified copies API and there is a CertifiedCopyItem class and it has the collection name as @Document(collection = "certified_copies"). In this should be @Document(collection = "scan_upon_demands")